src/beacon/drand.rs

```diff
@@ -75,17 +74,10 @@ impl BeaconSchedule {
         parent_epoch: ChainEpoch,
         prev: &BeaconEntry,
     ) -> Result<Vec<BeaconEntry>, anyhow::Error> {
-        let (cb_epoch, curr_beacon) = self.beacon_for_epoch(epoch)?;
```

Comment:
This aligns with https://github.com/filecoin-project/lotus/pull/11572/files#diff-587eaf0df6b60dbf741d19bbe39439f6f48ffe171e82a71c76b82484ba48f386R55